### DISCUSSION TITLE
Organized Layouts in Grouped Folders Not Working as expected

### DISCUSSION THREAD

**Discussion by HasanHatem** [vercel/next.js#60905] [Help]
### Summary

I have an issue when I have two or more grouped layouts.

![Frame 23](https://github.com/vercel/next.js/assets/16563153/7e8633f6-5032-438a-baec-216bf95040a9)

As you can see, I have described the page structure. **The problem** is that when I access the pricing page, it loads the Tools Layout, and the children do not appear on the page as well. I want it to load the Front Layout, but according to the documentation, it should load the nearest layout.

**The weird thing is that only the Home Page loads the Front Layout.**

I also tried to create multiple root layouts as mentioned here in the [Docs](https://nextjs.org/docs/app/building-your-application/routing/route-groups#creating-multiple-root-layouts), but it's not working. It still loads the Tools Layouts.

When I group all the tools routes in the tools folder, the problem is fixed. However, I want the tools to be in the main routes, not in subroutes.

Does anyone have solutions for this? Did I make any mistakes here?

The Main Root

```
import '../globals.css'
import { locales } from '@/navigation';
import { unstable_setRequestLocale } from 'next-intl/server';
import {notFound} from "next/navigation";

export function generateStaticParams() {
    return locales.map((locale) => ({locale}));
}

export default function RootLayout({children, params: {locale}}) {
    if (!locales.includes(locale)) notFound();
    unstable_setRequestLocale(locale);

    return (
        <html lang={locale} dir={locale === 'ar' ? 'rtl' : 'ltr'} suppressHydrationWarning={true}>
            <body>
                {children}
            </body>
        </html>
    )
}
```

**Comment by HasanHatem** [comment]
Any help guys?

**Comment by ShahriarKh** [comment]
Probably, you're facing this because route groups are not actual routes, thus `(front)/layout.jsx` and `(tools)/layout.jsx` are at the same level (both become `/layout.jsx`) and the latter overrides the former.

I'm not sure what I say is 100% true, but thought may help...
